<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Silo_dialog</class>
 <widget class="QDialog" name="Silo_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lvl_silo_l">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="capacity_silo_l">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lvl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upgrade_silo_butt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius:40px;
color: rgb(120, 60, 0);
background-color: rgb(255, 201, 66);</string>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>capacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="stock_silo_l">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stock</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
